### PR TITLE
Replace Pulsar with Space Acres Guide and Redirect Links

### PR DESCRIPTION
**Pull request by ImmaZoni**
This Pull Request introduces several key updates to our documentation:

1. **Removal of Pulsar**: As Pulsar is being sunset and phased out, this PR removes all references to Pulsar from our documentation.

2. **Introduction of Space Acres**: To fill the gap left by Pulsar, we have added a new guide titled 'Space Acres'. This serves as the new, preferred pathway for users, offering a streamlined, simplified farming experience to what Pulsar provided in the past. 

3. **Link Redirection**: All existing links that previously directed users to Pulsar pages have been updated to redirect to the relevant Space Acres pages, ensuring a seamless transition for users.

4. **Miscellaneous Link Fixes**: We have also taken this opportunity to fix various broken or outdated links in the 'Prerequisites' page, particularly those pertaining to the advanced-CLI and Space Acres guides.

### CODE REVIEW

**Comment by jim-counter** [comment]
Can we switch Space Acres to be "recommended" please @ImmaZoni?

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+## Get Started with Advanced CLI Installation
+
+## Step 1: Download the Space-Acres Executable
+
+<DownloadButtons />
+
+## Step 2: Install Space-Acres
+
+1. Open the downloaded Space-Acres executable to launch the Setup Wizard. This wizard will guide you through the installation process.
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:22]:
```suggestion
1. Open the downloaded Space-Acres installer to launch the Setup Wizard. This wizard will guide you through the installation process.
```

File: versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+## Get Started with Advanced CLI Installation
+
+## Step 1: Download the Space-Acres Executable
+
+<DownloadButtons />
+
+## Step 2: Install Space-Acres
+
+1. Open the downloaded Space-Acres executable to launch the Setup Wizard. This wizard will guide you through the installation process.
```
Comment (jim-counter) [inline versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx:22]:
```suggestion
1. Open the downloaded Space-Acres installer to launch the Setup Wizard. This wizard will guide you through the installation process.
```

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    - Farming
+    - GUI
+    - Executable
+---
+
+import DownloadButtons from './platforms/_download.mdx'
+
+## Get Started with Advanced CLI Installation
+
+## Step 1: Download the Space-Acres Executable
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:16]:
```suggestion
## Step 1: Download the Space-Acres Installer
```

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+keywords:
+    - Farmer
+    - Farming
+    - GUI
+    - Executable
+---
+
+import DownloadButtons from './platforms/_download.mdx'
+
+## Get Started with Advanced CLI Installation
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:14]:
```suggestion
## Get Started with Space Acres installation
```

File: versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+keywords:
+    - Farmer
+    - Farming
+    - GUI
+    - Executable
+---
+
+import DownloadButtons from './platforms/_download.mdx'
+
+## Get Started with Advanced CLI Installation
```
Comment (jim-counter) [inline versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx:14]:
```suggestion
## Get Started with Space Acres installation
```

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+    ![space-acres-install-4](/img/doc-imgs/space-acres/space-acres-install-4.png)
+
+5. Once the installation is complete, you will see a completion screen where you can choose to run Space Acres immediately by checking the box before clicking Finish.
+
+    ![space-acres-install-5](/img/doc-imgs/space-acres/space-acres-install-5.png)
+
+## Step 3: Initalize Space-Acres
+
+1. The welcome screen provides an overview of the necessary prerequisites for running Space Acres effectively. Make sure you meet all listed requirements before proceeding. For a comprehensive list of prerequisites and explanations, refer to our [prerequisites guide.](./space-acres-prerequisites.mdx)
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:44]: Prerequisite link will need changing when we merge them. Not reported in version-latest.

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+    ![space-acres-setup-4](/img/doc-imgs/space-acres/space-acres-setup-4.png)
+
+5. Finally, you will see the main interface of Space Acres, where your consensus node is running and your farmer is progressing along. Initial syncing & plotting may take some time to complete. 
+
+    ![space-acres-setup-5](/img/doc-imgs/space-acres/space-acres-setup-5.png)
+
+## Step 4: Start Farming
+
+Now that Space Acres is set up, your farming operation will begin automatically. Monitor the application for the status of your farming and the accrual of rewards.
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:78]: We should add a screenshot of what Space Acres looks like with a plotted farm. I will email one.

**Review by jim-counter — CHANGES_REQUESTED** [review]
I am going to suggest we move pre-requisites up a level so it sits under "Farming". There is only one item that is not applicable to SA and ACLI which could go on the install page.

We also need to make sure there is somewhere for those following Pulsar links to explain what has happened and next steps. Please see the Slack thread I've pinged you in @ImmaZoni. Thanks.

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx (hunk 6 above)
Comment (ImmaZoni) [reply docs/farming-&-staking/farming/space-acres/space-acres-install.mdx]: Fixed in Commit [49564d5](https://github.com/subspace/subspace-docs/pull/447/commits/49564d5c02530b74351695b23e275f6ff72ea835)

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx (hunk 7 above)
Comment (ImmaZoni) [reply docs/farming-&-staking/farming/space-acres/space-acres-install.mdx]: Fixed in Commit [49564d5](https://github.com/subspace/subspace-docs/pull/447/commits/49564d5c02530b74351695b23e275f6ff72ea835)

File: docs/farming-&-staking/farming/pulsar.mdx [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+In pursuit of delivering the simplest and most accessible farming experience, we are phasing out Pulsar. This decision honors our commitment to ease of use and development efficiency. Pulsar's sunset marks a shift in focus towards a more intuitive and robust GUI application, Space Acres, originally developed as a passion project by Nazar, now an Official Subspace Network application.
+
+### Transitioning to Space Acres GUI and Advanced CLI
+
+- **[Space Acres GUI Application](../farming/space-acres/space-acres-install)**: Space Acres represents the next step in our evolution, offering a graphical user interface that embodies our vision for simplicity without sacrificing the depth of farming features.
+
+- **[Advanced CLI](../farming/advanced-cli/cli-install)**: For those who prefer the command line, the Advanced CLI remains available, providing granular control over farming operations.
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/pulsar.mdx:24]:
```suggestion
- **[Advanced CLI](../farming/advanced-cli/cli-install)**: For those who prefer the command line or require greater configurability, the Advanced CLI remains available, providing granular control over farming operations.
```

File: versioned_docs/version-latest/farming-&-staking/farming/pulsar.mdx [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+In pursuit of delivering the simplest and most accessible farming experience, we are phasing out Pulsar. This decision honors our commitment to ease of use and development efficiency. Pulsar's sunset marks a shift in focus towards a more intuitive and robust GUI application, Space Acres, originally developed as a passion project by Nazar, now an Official Subspace Network application.
+
+### Transitioning to Space Acres GUI and Advanced CLI
+
+- **[Space Acres GUI Application](../farming/space-acres/space-acres-install)**: Space Acres represents the next step in our evolution, offering a graphical user interface that embodies our vision for simplicity without sacrificing the depth of farming features.
+
+- **[Advanced CLI](../farming/advanced-cli/cli-install)**: For those who prefer the command line, the Advanced CLI remains available, providing granular control over farming operations.
```
Comment (jim-counter) [inline versioned_docs/version-latest/farming-&-staking/farming/pulsar.mdx:24]:
```suggestion
- **[Advanced CLI](../farming/advanced-cli/cli-install)**: For those who prefer the command line or require greater configurability, the Advanced CLI remains available, providing granular control over farming operations.
```

**Review by jim-counter — CHANGES_REQUESTED** [review]
One small amendment please.

**Review by jim-counter — APPROVED** [review]
LGTM!

**Review by EmilFattakhov — COMMENTED** [review]
@ImmaZoni please don't merge it just yet, I need 15-20 minutes extra to go through it. thanks!

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+    - Farming
+    - GUI
+    - Executable
+---
+
+import DownloadButtons from './platforms/_download.mdx'
+
+## Get Started with Space Acres installation
+
+## Step 1: Download the Space-Acres Installer
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:16]: Installer -> installer

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+
+<DownloadButtons />
+
+## Step 2: Install Space-Acres
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:20]: We need to replace all instances of Space-Acres to Space Acres to be consistent.

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 12]
```diff
@@ -0,0 +1,107 @@
...
+    ![space-acres-setup-2](/img/doc-imgs/space-acres/space-acres-setup-2.png)
+
+3. (Optional) If you want to configure advanced settings, toggle the `Advanced configuration` section. This will allow you to set the following:
+    - Substrate Node P2P Port
+    - Subspace DSN P2P Port
+    - Enable Faster Networking (QUIC)
+
+    For more information on networking and its implications, refer to our [networking guide.](../additional-guides/networking)
+
+    ![space-acres-setup-3](/img/doc-imgs/space-acres/space-acres-setup-3.png)
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:66]: Sorry to be anal about it, but it feels unexpected to see a different wallet address and omitted farm size on this screenshot, compared to the previous one. It also makes the next image feel wrong.

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 13]
```diff
@@ -0,0 +1,107 @@
...
+
+5. Finally, you will see the main interface of Space Acres, where your consensus node is running and your farmer is progressing along. Initial syncing & plotting may take some time to complete. Below are examples of the various stages you can expect. 
+
+    a. Cache Syncing
+
+    ![space-acres-setup-5](/img/doc-imgs/space-acres/space-acres-setup-5.png) 
+
+    b. Initial Plotting
+
+    ![space-acres-setup-6](/img/doc-imgs/space-acres/space-acres-setup-6.png)
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:80]: I understand why those screenshots are different, but to me, it looks inconsistent to have a 10GB farm on one screenshot and a 2TB+ farm on the second one. It doesn't feel like a part of one process anymore.

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 14]
```diff
@@ -0,0 +1,107 @@
...
+    | Downloading       | ![#499249](https://via.placeholder.com/15/499249/000000?text=+) #499249 | Sector is in the process of downloading (animated) |
+    | Encoding          | ![#ff5365](https://via.placeholder.com/15/ff5365/000000?text=+) #ff5365 | Sector is encoding data (animated)   |
+    | Writing           | ![#9fc4ff](https://via.placeholder.com/15/9fc4ff/000000?text=+) #9fc4ff | Sector is writing data (animated)    |
+
+
+
+:::note Learn More About Syncing, Plotting & Farming
+
+If you would like to learn more about how the syncing, plotting and farming processes works on the Subspace Network, please refer to our Subnomicon links below.
+- [Synchronization](https://subnomicon.subspace.network/docs/network/network_protocols/#synchronization)
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:103]: Same comment about `-` here. Works fine as is, might let us down in the future.

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx (hunk 10 above)
[the remark this answers is not in the document]
Comment (ImmaZoni) [reply docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:16]: The current ## matches what we have in CLI Install, I prefer to keep them the same on the two documents imo, but curious your thoughts

Comment (ImmaZoni) [reply docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:16]: It is a title, so should remain capitalized, though with that being said "installation" should be capitalized aswell 😅

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx (hunk 13 above)
Comment (ImmaZoni) [reply docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:80]: Regarding screenshots, I will have to do them all from scratch to resolve these issues, while I agree Its going to take some time to get a full "inline" set of screenshots as i will have to wait for complete node sync etc.

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx (hunk 10 above)
[the remark this answers is not in the document]
Comment (EmilFattakhov) [reply docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:16]: You're right, wow, it looks so mich larger here. Okay, let's keep this unchanged for consistency then.

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx (hunk 14 above)
Comment (ImmaZoni) [reply docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:103]: Fixed in https://github.com/subspace/subspace-docs/pull/448

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx (hunk 11 above)
Comment (ImmaZoni) [reply docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:20]: Fixed in https://github.com/subspace/subspace-docs/pull/448

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx (hunk 10 above)
Comment (ImmaZoni) [reply docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:16]: Fixed in https://github.com/subspace/subspace-docs/pull/448

File: docs/farming-&-staking/farming/pulsar.mdx [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+
+- **[Subspace Forum](https://forum.subspace.network)**: Our forum is a hub for discussion and community assistance.
+- **[Subspace Discord](https://discord.subspace.network)**: Join our Discord for real-time support and to connect with fellow farmers.
+
+We are grateful for your adaptability and support as we navigate this change together.
+
+:::note Learn More About Syncing, Plotting & Farming
+
+For more information about the principles of farming on the Subspace Network, you can still refer to the Subnomicon links below:
+- [Synchronization](https://subnomicon.subspace.network/docs/network/network_protocols/#synchronization)
```
[the remark this answers is not in the document]
Comment (ImmaZoni) [reply docs/farming-&-staking/farming/pulsar.mdx:44]: Fixed in https://github.com/subspace/subspace-docs/pull/448

File: docs/farming-&-staking/farming/prerequisites.mdx [hunk 16]
```diff
@@ -77,3 +76,17 @@ In order to disable **CoW** on **BTRFS**, run the command `chattr +C /your_farme
...
+
+:::caution  Windows No Output Bug
+If you face an error where the node outputs nothing and no error code is given it is likely you just need to install the latest Visual C++ Redistributable package [here](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170)
+:::
+
+:::caution Linux `libcomp.so.1` error.
+If you encounter an error related to `libgomp.so.1`, install the `libgomp1` library with `sudo apt-get install libgomp1`.
+:::
+
+:::caution Linux File Descriptor Limit
```
[the remark this answers is not in the document]
Comment (ImmaZoni) [reply docs/farming-&-staking/farming/prerequisites.mdx:90]: Fixed in https://github.com/subspace/subspace-docs/pull/448

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx (hunk 13 above)
Comment (EmilFattakhov) [reply docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:80]: Yeah, and with each drastic change in the UI, they will have to be changed as well. 

Let's create a kanban board task to keep that in mind next time we're spinning up another SA node.

**Pull request merged**